frontend/lib/src/components/widgets/DataFrame/hooks/useCustomTheme.ts

```diff
@@ -75,9 +89,10 @@ function useCustomTheme(): CustomThemeReturn {
       textLight: theme.colors.fadedText40,
       textBubble: theme.colors.fadedText60,
       bgCell: theme.colors.bgColor,
-      bgCellMedium: theme.colors.bgColor, // uses same as bgCell to always have the same background color
-      cellHorizontalPadding: 8,
-      cellVerticalPadding: 3,
+      // uses same as bgCell to always have the same background color:
+      bgCellMedium: theme.colors.bgColor,
+      cellHorizontalPadding: Math.round(convertRemToPx(theme.spacing.sm)),
+      cellVerticalPadding: Math.round(convertRemToPx("0.1875rem")),
```

Comment:
These values will likely be cleaned up in a follow-up PR. The goal in this PR was mainly refactoring it without triggering visual changes.